rest_api/VulnMgmtService.adoc

```diff
@@ -0,0 +1,3129 @@
+| -
+| null
+|
+
+|===
+
+
+=== Return Type
+
+<<Stream_result_of_v1VulnMgmtExportWorkloadsResponse>>
+
+
+=== Content Type
+
+* application/json
+
+=== Responses
+
+.HTTP Response Codes
+[cols="2,3,1"]
+|===
+| Code | Message | Datatype
+
+
+| 200
+| A successful response.(streaming responses)
+|  <<Stream_result_of_v1VulnMgmtExportWorkloadsResponse>>
+
+
+| 0
+| An unexpected error response.
+|  <<RuntimeError>>
+
+|===
+
+=== Samples
+
+
+
+
+
+
+
+
+
+ifdef::internal-generation[]
+=== Implementation
+
+
+
+endif::internal-generation[]
+
+
+[id="common-object-reference"]
+== Common object reference
+
+
+
+[#CVSSV2AccessComplexity]
+=== _CVSSV2AccessComplexity_ 
+
+
+
+
+
+
+[.fields-CVSSV2AccessComplexity]
+[cols="1"]
+|===
+| Enum Values
+
+| ACCESS_HIGH
+| ACCESS_MEDIUM
+| ACCESS_LOW
+
+|===
+
+
+[#CVSSV2Authentication]
+=== _CVSSV2Authentication_ 
+
+
+
+
+
+
+[.fields-CVSSV2Authentication]
+[cols="1"]
+|===
+| Enum Values
+
+| AUTH_MULTIPLE
+| AUTH_SINGLE
+| AUTH_NONE
+
+|===
+
+
+[#CVSSV3Complexity]
+=== _CVSSV3Complexity_ 
+
+
+
+
+
+
+[.fields-CVSSV3Complexity]
+[cols="1"]
+|===
+| Enum Values
+
+| COMPLEXITY_LOW
+| COMPLEXITY_HIGH
+
+|===
+
+
+[#CVSSV3Privileges]
+=== _CVSSV3Privileges_ 
+
+
+
+
+
+
+[.fields-CVSSV3Privileges]
+[cols="1"]
+|===
+| Enum Values
+
+| PRIVILEGE_NONE
+| PRIVILEGE_LOW
+| PRIVILEGE_HIGH
+
+|===
+
+
+[#CVSSV3UserInteraction]
+=== _CVSSV3UserInteraction_ 
+
+
+
+
+
+
+[.fields-CVSSV3UserInteraction]
+[cols="1"]
+|===
+| Enum Values
+
+| UI_NONE
+| UI_REQUIRED
+
+|===
+
+
+[#ContainerConfigEnvironmentConfig]
+=== _ContainerConfigEnvironmentConfig_ 
+
+
+
+
+[.fields-ContainerConfigEnvironmentConfig]
+[cols="2,1,1,2,4,1"]
+|===
+| Field Name| Required| Nullable | Type| Description | Format
+
+| key
+| 
+| 
+|   String  
+| 
+|     
+
+| value
+| 
+| 
+|   String  
+| 
+|     
+
+| envVarSource
+| 
+| 
+|  <<EnvironmentConfigEnvVarSource>>  
+| 
+|    UNSET, RAW, SECRET_KEY, CONFIG_MAP_KEY, FIELD, RESOURCE_FIELD, UNKNOWN,  
+
+|===
+
+
+
+[#EmbeddedVulnerabilityVulnerabilityType]
+=== _EmbeddedVulnerabilityVulnerabilityType_ 
+
+
+
+
+
+
+[.fields-EmbeddedVulnerabilityVulnerabilityType]
+[cols="1"]
+|===
+| Enum Values
+
+| UNKNOWN_VULNERABILITY
+| IMAGE_VULNERABILITY
+| K8S_VULNERABILITY
+| ISTIO_VULNERABILITY
+| NODE_VULNERABILITY
+| OPENSHIFT_VULNERABILITY
+
+|===
+
+
+[#EnvironmentConfigEnvVarSource]
+=== _EnvironmentConfigEnvVarSource_ For any update to EnvVarSource, please also update 'ui/src/messages/common.js'
```

Comment:
I feel like this might be an error with parsing something and the "For any update" part shouldn't be included here?
![Screenshot 2024-09-25 at 3 25 51 PM](https://github.com/user-attachments/assets/476cc421-21ae-45c4-8c5a-5fe1b37e608d)